plugins/inputs/mysql/README.md

```diff
@@ -126,6 +126,9 @@ in mySQL configuration. See the performance schema [quick start][quick-start].
   # tls_key = "/etc/telegraf/key.pem"
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
+
+  ## Aggregate table_io_waits
+  # aggregate_table_io_waits = true
```

Comment:
What's the reasoning for this change? It doesn't seem relevant to CCR

Reply:
its not in the CCR commit, its from another commit, just included in the PR

Reply:
I want to base all those commits on top of "1.23.4-aiven"

Reply:
https://github.com/aiven/telegraf/pull/76/commits/b767cc6dd22a229456cd1ae43a2ffcd17de3981c this is the only new commit